preprocessy/outliers/_handleoutlier.py

```diff
@@ -56,8 +56,7 @@ def __init__(self, train_df, params):
             self.third_quartile = params["q3"]
 
     def __repr__(self):
-        params_default = dict(([], {}))
-        return f"HandleOutlier(train_df=None,params={params_default})"
+        return f"HandleOutlier(train_df=None,params=None)"
```

Comment:
I guess you forgot to resolve this: https://github.com/preprocessy/preprocessy/pull/37#discussion_r565840028